benchmarks/ALCE/ELI5/README.md

```diff
@@ -0,0 +1,104 @@
+            "summary": "Snapchat is aiming to raise $500 million with a valuation of $16 billion to $19 billion, making it the third most highly valued tech start-up backed by venture capitalists. Other highly valued companies include Xiaomi at $45 billion and Uber at about $40 billion. Snapchat was previously valued at $10 billion, and some of its investors include Benchmark, Lightspeed Venture Partners, and Kleiner Perkins Caufield. The article does not discuss whether venture capitalists expect repayment within a certain timeframe.",
+            "extraction": "Venture capitalists invest in startups with the expectation of a big payout down the road, either through an initial public offering or an acquisition. This means that the investors expect the company to eventually become profitable and generate returns on their investment. However, in the case of firms like Snapchat and Uber, they are valued highly because of their potential for future growth and dominance in their respective markets, rather than current profitability. Therefore, venture capitalists may not expect repayment within a specific number of years but instead expect a significant"
+        },
+        {
+            "title": "What Are Venture Capital Investments? \u2013 DollarsAndSense.my",
+            "text": "Ever wondered how highly valued technology giants like Google and Facebook were able to grow so fast and pay their employees so well in such a short amount of time, or how still growing start-ups like Uber are able to lose 1.2 billion US dollars in just the first half of this year alone and still command a valuation upwards of 50 billion US dollars? The answer lies with a special category of investment activity known as venture capital. Venture capitalists are professional investors who invest in a number of highly scalable high-risk technology ventures hoping to make a multi-fold",
+            "url": "http://dollarsandsense.my/what-are-venture-capital-investments/",
+            "summary": "Venture capitalists invest in highly scalable high-risk technology ventures, such as Snapchat and Uber, hoping to make a multi-fold return on their investment. This explains how firms can be valued highly despite not making a profit.",
+            "extraction": "The reason why firms like Uber can command such high valuations despite not making a profit is due to venture capital investments. Venture capitalists are professional investors who invest in high-risk technology ventures with the hopes of making a multi-fold return on their investment. Therefore, they do expect some form of repayment within a number of years but are willing to take on the risk in exchange for the potential high returns."
+        },
+        ...
+    ]
+}
+```
+
+## Metrics
+
+In the original paper, the authors evaluated four metrics on the ELI5 dataset: MAUVE, Claim Recall, Citation Recall, and Citation Precision. 
```

Comment:
To evaluate performace on 'Eli5-ALCE' dataset, we use following metrics as default: 
[Citation Precision](./rageval/metrics/_answer_citation_precision.py), used in paper [1]...
Claim Recall
Citation Recall
MAUVE
ROUGH_L
F1
.......